### DISCUSSION TITLE
Config global state for standalone Keycloak 22.0.5

### DISCUSSION THREAD

**Discussion by phoenix-ua** [keycloak/keycloak#26095] [Q&A]
Greetings,

Our team uses a single standalone on-premise Keycloak 8.0.1 instance and we're migrating it to Keycloak 22.0.5. We noticed a couple of warning messages during Keycloak boot and the one I'm worried about is:
```
WARN  [org.infinispan.CONFIG] (keycloak-cache-init) ISPN000569: Unable to persist Infinispan internal caches as no global state enabled
```

Is this warning an indicator of any problem that has to be fixed in my specific case with a single Keycloak instance? I never faced a 'global state' notion in Keycloak before.

I consulted with [Configuring Keycloak for production](https://www.keycloak.org/server/configuration-production) and [Configuring distributed caches](https://www.keycloak.org/server/caching) docs but only found recommendations for multi-node Keycloak clusters and nothing about standalone installations. Maybe there are examples of standalone Keycloak configuration without such warning messages or I missed something?

I will be grateful for any assistance or hints

**Comment by codespearhead** [comment]
Can you close this discussion in favor of #29384 ?

That one has more troubleshooting data.

**Comment by pruivo** [comment]
The warning can be safely ignored because Keycloak does not use any Infinispan feature that requires the global state.
